spec/unit/type/file/content_spec.rb

```diff
@@ -13,6 +13,8 @@
   before do
     File.open(filename, 'w') {|f| f.write "initial file content"}
     described_class.stubs(:standalone?).returns(false)
+    Puppet::Type.type(:file).any_instance.stubs(:file).returns('my/file.pp')
+    Puppet::Type.type(:file).any_instance.stubs(:line).returns 5
```

Comment:
What happens if you don't add this stub here? I would assume we never hit the code you added above unless we're actually getting a deprecation warning?

Reply:
Many tests in this file are testing that you can explicitly put a checksum in the content field. Those tests will trigger deprecation warnings. 

The warnings are also getting triggered when "testing whether content is in sync" with mtime and and ctime. I'm still working to see if that is part of the set up of the test, or if  the deprecation warning is being triggered where it shouldn't be.

Reply:
The warnings were just part of the test set up, however it made sense to remove the stubs and add a stub for the source property instead to more realistically mimic the application.